extensions/commands/cci/cmd_list_v2_ready.py

```diff
@@ -0,0 +1,114 @@
+import json
+import os
+import yaml
+
+import conan.api.conan_api
+from conan.api.model import ListPattern
+from conan.api.output import ConanOutput
+from conan.cli.command import conan_command, OnceArgument
+from conan.errors import ConanException
+
+
+def output_json(results):
+    print(json.dumps(results, indent=2))
+
+
+@conan_command(group="Conan Center Index", formatters={"json": output_json})
+def list_v2_ready(conan_api: conan.api.conan_api.ConanAPI, parser, *args):
```

Comment:
I understand for us this is useful but how applicable is this to a wider audience and what happened when we required v2 support?

This wont help find all the old stale versions which users will see but not be able to use 🤔

Reply:
Good last point, we can always iterate over the commands. This was needed in c3i so it got  split into its own PR (And merged without tests, we were in a hurry 😢) The question of how this command can bring value to the users is a good one, and we should think about it, definitely :)